src/pages/uxp/reference-js/Modules/os/OS.md

```diff
@@ -33,3 +28,40 @@ Gets the platform architecture we are running on (eg. "x32, x64, x86_64 etc")
 
 **Returns**: `string` - the string representing the architecture  
 
+
+<a name="os-cpus" id="os-cpus"></a>
+
+## cpus()
+Gets the platform cpu information we are running on (eg. "{'Intel(R) Core(TM) i9-8950HK CPU @ 2.90GHz', 2900}")
+
+**Returns**: `array` - the array of objects containing information about each logical CPU core
```

Comment:
`array` of what? (and if it's `Array<any>`, so be it, but don't leave me hanging like that 😉)

Reply:
🟡